htdocs/core/modules/modProductBatch.class.php

```diff
@@ -65,7 +65,7 @@ public function __construct($db)
 		$this->dirs = array();
 
 		// Config pages. Put here list of php page, stored into productdluo/admin directory, to use to setup module.
-		$this->config_page_url = array("product_lot_extrafields.php@product");
```

Comment:
Why removing this setup page ?

Reply:
i added the product_lod.php config page that works with product_lot_admin_prepare_head ; i cant see why we have to define each admin tab here because it works with the main admin page ?

Reply:
Ok, you're right.
the product_lot_admin_prepare_head contains already code to add the product_lot_extrafields tag.

The config_page_url is a feature for external module. But here, because we add this into core, you can add directly the declaration of $head for the new tab into 
product_lot_admin_prepare_head()
and keep  $this->config_page_url as an empty array

Reply:
i(ve just tried and it does not work if you leave the config_page_url array empty.